src/utils/sendTaskUpdates.ts

```diff
@@ -0,0 +1,24 @@
+import { env } from "../typeDefinitions/default.types";
+import config from "../../config/config";
+
+export async function sendTaskUpdate(
+  completed: string,
+  planned: string,
+  blockers: string,
+  env: env
+): Promise<void> {
+  const formattedString = `**Completed**: ${completed}\n\n**Planned**: ${planned}\n\n**Blockers**: ${blockers}`;
+  const bodyObj = {
+    content: formattedString,
+  };
+  const url = config(env).TRACKING_CHANNEL_URL;
+  await fetch(url, {
+    method: "POST",
+    body: JSON.stringify(bodyObj),
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bot ${env.DISCORD_TOKEN}`,
+    },
+  });
+  return;
+}
```

Comment:
`sendTaskUpdate` function does not handle errors from the fetch call. It's important to handle potential network errors, such as timeouts or failed requests, to provide better resilience to the application.

Reply:
Please check the error handling is done in the taskUpdateHandler.